core/chain/abci.go

```diff
@@ -113,19 +140,18 @@ func (app *CoreApplication) FinalizeBlock(ctx context.Context, req *abcitypes.Fi
 				txs[i] = &abcitypes.ExecTxResult{Code: 2}
 			}
 			txs[i] = &abcitypes.ExecTxResult{Code: abcitypes.CodeTypeOK}
-			continue
```

Comment:
after adding the new cases here: https://github.com/AudiusProject/audius-protocol/blob/main/core/chain/finalize.go#L10-L17
let's bring this `continue` back so we don't fall into the kv case. We'll remove the kvstore in a subsequent pr. it's served it's purpose